build/platform.simulator.linux.mak

```diff
@@ -1,2 +1,2 @@
 TOOLCHAIN = host-gcc
-EXE = bin
+EXE = elf
```

Comment:
Why is this needed?

Reply:
It's not mandatory, but since we generate ELF files under Linux we might as well use the correct file extension.